LondonBritishLibrary/orient/BLorient7945.xml

```diff
@@ -0,0 +1,162 @@
+                                <foreign xml:lang="am"><hi rend="rubric">አምስት፡ ክንድ፡ ርጅም፡ የምያህል፡</hi></foreign>.</note>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">92</measure>
+                                    <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                                        <height>232</height>
+                                        <width>190</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Original pagination by the author from 1 to 180.</foliation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="23"><locus target="#2v"/><locus from="45v" to="50r"/>
+                                    <locus from="89v" to="180r"/></layout>
+                                <layout columns="2" writtenLines="23"><locus from="3r" to="44r"/><locus from="50v" to="89r"/></layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Latin">
+                                <desc>Hand of Captain T. C. S. Speedy, written in black and red ink.</desc>
+                                <date notBefore="1867" notAfter="1868" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1v"/>
+                                    <desc type="ScribalSignature">A note possibly by the author <persName role="author" ref="PRS8943Speedy">
+                                        <roleName type="rank">Captain</roleName>T. C. S. Speedy</persName> himself.</desc>
+                                    <q xml:lang="en">Vocabulary of the Amharic Language by <persName role="author" ref="PRS000000000000">
+                                        <roleName type="rank">Captain</roleName>T. C. S. Speedy</persName>, Interpreter on the Staff of 
+                                        <persName ref="PRS7484Napier"><roleName type="title">Lord</roleName> Napier</persName>
+                                        on <placeName ref="LOC4547Maqdal">Magdala</placeName>, during the Abyssinian Expedition of 
+                                        <date notBefore="1867" notAfter="1868">1867-1868</date></q>
```

Comment:
If this is a quotation from the note, I would quote it like it is given by Strelcyn

Reply:
The oxygen does not accept the sign "&". Therefore I cannot represent it in the record. The only thing, I can do, is to add "et" instead of "&". I think in this place it is dispensable. "during the Abyssinian Expedition of 1867&/1868". (But I can change "-" to "/".

Reply:
I think you need to type &amp; to produce "&", but you are right, it's not a matter of life and death :)

Reply:
> I think you need to type & to produce "&", but you are right, it's not a matter of life and death :)

You need to use 'Code' (see above) to show a xml-code:

<img width="197" alt="Screenshot 2024_CodeII" src="https://github.com/user-attachments/assets/dac224ae-e86b-4ac0-b19a-be40880bf75c">

Reply:
`&amp;` – thank you! :)

Reply:
You mean like this:
`<date notBefore="1867" notAfter="1868">1867&amp;/1868</date>`
Thank you! I did not find this trick in the Guidelines.